### PR TITLE
Allows for disabling the server side API requests

### PR DESCRIPTION
**Pull request by abea**
With significant changes to the Facebook API, I want to provide the opportunity to disable the server side requests to allow more complete client-side override.

### CODE REVIEW

**Review by boutell — DISMISSED** [review]
Chuck a note in the README. Maybe call this apiDisabled.

**Comment by boutell** [comment]
Otherwise fine

**Comment by abea** [comment]
@boutell Updated and ready to quiet those staging notifications.

File: index.js [hunk 1]
```diff
@@ -171,6 +171,7 @@ function Construct(options, callback) {
       return res.send(results);
     });
   });
```
Comment (boutell) [inline index.js]: Is there a double close of `})` going on here?

**Review by boutell — CHANGES_REQUESTED** [review]

**Review by boutell — APPROVED** [review]

**Pull request merged**